RELEASE_NOTES.md

```diff
@@ -9,6 +9,16 @@ incompatible changes have been made in that release.
 For a list of all the changes in a particular release, please refer to the changelog file -
 https://github.com/scalyr/scalyr-agent-2/blob/master/CHANGELOG.md.
 
+## 2.2.1 "TBD"
+
+* This release significantly changes the Agent's `deb` and `rpm` packages. Agent now is shipped with its own, independent 
+  version of the Python interpreter and does not rely on system Python. The upgrade process to the new packages has to be 
+  seamless for the majority of cases, but if you use custom monitors that require additional libraries, you will need to
+  specify those requirements in new agent's configuration.
+  
+  After upgrading to a new version, add required libraries to the file `/opt/scalyr-agent-2-dependencies/etc/additional-requirements.txt` 
```

Comment:
We probably also need some docs on dataset.com/help/ on that. So we should probably also start working on that (open ticket, etc.) and then probably also involve @peteryork in the future.

Reply:
Yeah, I've already tried to find the most appropriate place for it but not sure where we can put that. I guess the closes thing has to be this one https://app.scalyr.com/help/scalyr-agent#plugins

Reply:
PS: But, I guess, since we kinda claim that those tested distros are officially supported, maybe we should take an effort and do this for all of them. WDYT?

Reply:
Sorry, this is a wrong thread.